code-challenges.js

```diff
@@ -40,7 +98,23 @@ var testString1 = "albatross"
 var testString2 = "jabberwocky"
 // Expected output: "jAbBeRwOcKy"
 
+//create a function that takes in a string
+const rudeType = (string) =>{
+    //split the string into an array of characters and store it in a variable
+    let strArr = string.split("")
+    //for every other letter in array toUpperCase it.
+    return strArr.map((value, index) =>{
+        if (index % 2 === 0){
+            return value
+        } else {
+            return value.toUpperCase()
+        }
+    //join the array back together into a string.
+    }).join("")
 
+}
+// console.log(rudeType(testString1));
+// console.log(rudeType(testString2));
```

Comment:
"rudeType" Hahaha 😆 thats great! Excellent solution! Look at you chainin' those methods. YEAH!